kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.0.3
+    0.0.4
 
 owners:
     [jjeffryes, tgu2]
```

Comment:
you can add yourself as owner so that you can register module

Reply:
done